content/en/docs/instrumentation/js/instrumentation.md

```diff
@@ -252,37 +315,36 @@ function someFunction(spanToLinkFrom) {
 
 ## Span Status
 
-A status can be set on a span, typically to indicate that it did not complete
-successuflly - `SpanStatusCode.ERROR`. In rare situations, you may wish to
-override this with `SpanStatusCode.OK`. But don't set the status to `OK`
-each time a span successfully completes.
+A [status](/docs/concepts/signals/traces/#span-status) can be set on a span,
+typically used to specify that a span has not completed successfully -
+`SpanStatusCode.ERROR`.
 
 The status can be set at any time before the span is finished:
 
 ```javascript
-function doWork(parent) {
-  const ctx = opentelemetry.trace.setSpan(
-    opentelemetry.context.active(),
-    parent
-  );
-  const span = tracer.startSpan('doWork', undefined, ctx);
-
-  for (let i = 0; i <= Math.floor(Math.random() * 40000000); i += 1) {
-    if (i > 10000) {
-      span.setStatus({
-        code: opentelemetry.SpanStatusCode.ERROR,
-        message: 'Error'
-      });
-    }
+const span = tracer.startSpan('doWork', undefined, ctx);
+
+for (let i = 0; i <= Math.floor(Math.random() * 40000000); i += 1) {
+  if (i > 10000) {
+    span.setStatus({
+      code: opentelemetry.SpanStatusCode.ERROR,
+      message: 'Error'
+    });
   }
-
-  span.end();
 }
+
+span.end();
 ```
 
+By default, the status for all spans is `Unset` rather than `Ok`. It is
+typically the job of another component in your telemetry pipeline to interpret
+the `Unset` status of a span, so it's best not to override this unless you're
```

Comment:
This statement sounds to me like suggesting users to not set the status code OK. Am I understanding this correctly?

Reply:
Yep. It's at least my understanding that unless you're an instrumentation library author, you shouldn't set the status to `Ok`, but leave it as `Unset` for a processor or backend to interpret.

Reply:
Could you elaborate on why that is?  Wouldn't the application dev know whether the span completed or not?